source/nodejs/adaptivecards-site/themes/adaptivecards/layout/_partial/header.ejs

```diff
@@ -457,9 +457,17 @@
         <div class="action-bar">
             <ul class="action-list">
                 <li>
-                    <a id="feedback-button" class="button" data-bi-name="feedback" tabindex="0" focusable="true">
+                    <a href="javascript:invokeFeedback();" class="button" data-bi-name="feedback" tabindex="0" focusable="true">
                         <span class="docon docon-comment-lines" aria-hidden="true"></span> <span class="action-item-text"><%= site.data.banner.en.feedback %></span>
                     </a>
+                    <script>
```

Comment:
@paulcam206 My only hesitation here is how it will interact with UHF if we get that working. What do you think?

We could also make a JS file within `source/js` and import the script if that's better.

Reply:
@paulcam206 done!